mangoglass/AgeOfGarb
# Added MapCreator_alvaro prefabs and with implemented NPC AI that move…

… towards a trashcan.

This is not finished yet!

TO TEST:
Just drag and drop the MapCreator_alvaro prefab into a scene and press play! Make sure that all the dependencies are to the scripts are present.
